assets/css/src/components-public/_public.scss

```diff
@@ -157,3 +157,36 @@ $form-columns-space-between: 20px;
     @include maxWidthColumn;
   }
 }
+
+div.mailpoet_form_popup {
+  background-color: white;
+  border-radius: 10px;
+  box-shadow: 0 4px 35px 0 rgba(195, 65, 2, .2);
+  display: none;
+  height: 240px;
```

Comment:
It can't have fixed height, because there can be many inputs. 

<img width="525" alt="Screenshot 2020-03-25 19 03 56" src="https://user-images.githubusercontent.com/470616/77570060-7cd56e00-6ecb-11ea-9bae-302769a7866c.png">